hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/OperatingSystemMetricSet.java

```diff
@@ -72,6 +72,11 @@ public double get(OperatingSystemMXBean bean) {
     // This method doesn't depend on the OperatingSystemMXBean so it can be tested. Due to not knowing
     // the exact OperatingSystemMXBean class it is very difficult to get this class tested.
     static void registerMethod(MetricsRegistry metricsRegistry, Object osBean, String methodName, String name) {
+        registerMethod(metricsRegistry, osBean, methodName, name, 1);
+    }
+
+    private static void registerMethod(MetricsRegistry metricsRegistry, Object osBean, String methodName,
+            String name, final long multiplier) {
```

Comment:
Do we need this `final` here?

Reply:
Yes, unless you use Java 8 compiler.

Reply:
Ah, now I see it. Was just a bit difficult to spot on GitHub :)